module/os/linux/zfs/vdev_object_store.c

```diff
@@ -274,7 +295,12 @@ agent_request(vdev_object_store_t *vos, nvlist_t *nv)
 		    (int)total_size, (int)sent);
 	}
 
+	if (zio_injection_enabled) {
+		zfs_dbgmsg("%s INJECTION after to send", tag);
```

Comment:
```suggestion
		zfs_dbgmsg("%s INJECTION after send", tag);
```
or `subsequent to send`